### ISSUE TITLE
IE 11 legend toggle

### ISSUE THREAD

**Issue by vlarkovich** [c3js/c3#2528]
**Steps to reproduce:**

1. Open any page with c3 chart in IE 11, for example [area chart](https://c3js.org/samples/chart_area.html) or [pie chart](https://c3js.org/samples/chart_pie.html)
2. Try to toggle any item on legend
3. See the chart with hidden item
![image](https://user-images.githubusercontent.com/43816231/48606008-722dcf00-e98f-11e8-9fff-f7de5f8f39dd.png)
 Or
![image](https://user-images.githubusercontent.com/43816231/48606094-b28d4d00-e98f-11e8-9f31-b195d3e806af.png)

**Expected result:** 
Data should not dissapear from chart.

Maybe it is because IE doesn't support `display: initial` css style and when api.show is called it doesn't remove `display: none` that was added in api.hide
https://github.com/c3js/c3/blob/9f546012615afa9b2c2fb85f36cbe7332cc374f7/src/api.show.js#L13

* **C3 version**: 0.6.8
* **D3 version**: 5.6.0
* **Browser**: IE 11
* **OS**: win10 x64

**Comment by SovietAlex** [comment]
This was already mentioned before. 
[(https://github.com/c3js/c3/issues/2486)]

Just change
.style('display', 'initial', 'important') 
to
.style('display', 'block', 'important')

it works in all browsers after this

**Comment by vlarkovich** [comment]
@SovietAlex, thank you, I've already fixed it locally, but I think that this bug has to be fixed in the main version of c3js, because this is a critical error.

**Comment by snakada** [comment]
Hi, it might be a bit tricky, but I solved it by writing the following in CSS:

```css
.c3-target[style*="opacity: 1"] {
    display: block !important;
}
```

**Comment by digEmAll** [comment]
I agree with @vlarkovich this should be included in the main version

**Comment by GDFaber** [comment]
I'd like to make a PR ready for this, but it may take some time as this would be my first PR for this project and I have to set up a few things according to the contribution guidelines first.

**Comment by GDFaber** [comment]
PR at #2741
